samples/typescript-express/.env.example

```
@@ -1,3 +1,5 @@
 METRIPORT_API_KEY="..."
 METRIPORT_WH_KEY="..."
 FACILITY_ID="..."
+# Keep this set to false in order to hit the Metriport sandbox environment.
+IS_PROD="false"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)